cccs-build/superset/Dockerfile

```diff
@@ -1,8 +1,7 @@
 # Vault CA container import
 ARG VAULT_CA_CONTAINER=uchimera.azurecr.io/cccs/hogwarts/vault-ca:master_2921_22315d60
 FROM $VAULT_CA_CONTAINER AS vault_ca
-FROM uchimera.azurecr.io/cccs/superset-base:cccs-2.0_20220823171152_b4666
-
+FROM uchimera.azurecr.io/cccs/superset-base:fix_cccs-2.0_20220830192042_b4757
```

Comment:
Do you really want to update the image?

Reply:
No! but im just using it to make stuff I can pop on udev

Reply:
Right, in that case it lgtm!